packages/validators-util/src/isInteger/isInteger.ts

```diff
@@ -0,0 +1,5 @@
+export const INTEGER_REGEX = /^-?[\s\d]+$/;
+
+export const isInteger = (value: string): boolean => {
+    return INTEGER_REGEX.test(value);
+};
```

Comment:
Skulle funksjonen også godtatt `number`?

```suggestion
export const isInteger = (value: string | number): boolean => {
    return INTEGER_REGEX.test(String(value));
};
```